### PR TITLE
Doc2vec processing

### PR DESCRIPTION
**Pull request by lindseydew**
Adds a method to process a CAPI dump in the format with urls and docs. 

@hmgibson23 if you could lemme know if the dump is finished we can test this with a real file and see how the performant it is.

### CODE REVIEW

File: suggested-content/word-2-vec-service.py [hunk 1]
```diff
@@ -27,6 +31,14 @@ def documentCheckPhrases():
     response = checkPhrases(grams)
     return jsonify(results = response)
 
+@app.route('/doc', methods=['POST'])
+def similarDocs():
+    doc = request.json["doc"]
```
Comment (hmgibson23) [inline suggested-content/word-2-vec-service.py:36]: It's not going to come in json format anymore. Instead it's one line with the url, line break, then the text content and another line break to separate the documents. I'll upload a sample of the file to S3

Comment (lindseydew) [inline suggested-content/word-2-vec-service.py:36]: Yeah this is an endpoint to process to a POST request (from composer say) which contains a body of text and can be analysed against our documents in memory. The method for processing the CAPI file is called loadDocuments and this should expect a file with url, break, text, break. We can test and modify if necessary when we have a sample file.

Comment (hmgibson23) [inline suggested-content/word-2-vec-service.py:36]: Oh sorry - I was spacing out!

On 11 November 2015 at 14:01, Lindsey Dew notifications@github.com wrote:

> In suggested-content/word-2-vec-service.py
> https://github.com/guardian/proposer-suggested-content/pull/27#discussion_r44534288
> :
> 
> > @@ -27,6 +31,14 @@ def documentCheckPhrases():
> >      response = checkPhrases(grams)
> >      return jsonify(results = response)
> > 
> > +@app.route('/doc', methods=['POST'])
> > +def similarDocs():
> > -    doc = request.json["doc"]
> 
> Yeah this is an endpoint to process to a POST request (from composer say)
> which contains a body of text and can be analysed against our documents in
> memory. The method for processing the CAPI file is called loadDocuments
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/guardian/proposer-suggested-content/pull/27/files#r44534288
> .

## 

---

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.

Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Comment by hmgibson23** [comment]
:+1:

**Pull request merged**